src/cli/commands/build/build.ts

```diff
@@ -104,6 +107,12 @@ export async function build(options: SWACLIConfig) {
     logger.log(`Building api with ${chalk.green(apiBuildCommand)} in ${chalk.dim(apiLocation)} ...`);
     runCommand(apiBuildCommand, apiLocation!);
   }
+  const end = new Date().getTime();
+
+  collectTelemetryEvent(TELEMETRY_EVENTS.Build, {
+    appFramework: projectConfig?.name?.split(", with")[0]!,
+    duration: (end - start).toLocaleString(),
```

Comment:
Why are we converting to LocaleString.. it will be different for different locales right.. so do we want different duration format in our logs ?